Dockerfile

```diff
@@ -17,5 +17,4 @@ RUN apt-get update && apt-get install -y wget --no-install-recommends \
   --no-install-recommends \
   && rm -rf /var/lib/apt/lists/* \
   && rm -rf /src/*.deb \
-  # Intall node-gyp in the base image to get around parallel installation issues
```

Comment:
this comment was causing the line below it `npx node-gyp@4 install` to not execute

Reply:
Does this cause a parse error then?  May be able to use postfix syntax instead if we want to keep this comment around.